dev/integration_tests/native_driver_test/android/app/src/main/kotlin/com/example/native_driver_test/MainActivity.kt

```diff
@@ -45,5 +47,15 @@ class MainActivity : FlutterActivity() {
         windowInsetsController.systemBarsBehavior = WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
         windowInsetsController.hide(WindowInsetsCompat.Type.systemBars())
         actionBar?.hide()
+
+        // The default behavior on SDK level 34 and below is for display cutouts to be consumed
```

Comment:
But the version below is `>= 30`?

Reply:
`WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_ALWAYS` was introduced in 30, and as of 35 it is essentially the default behavior. So for SDKs 30<=x<=34 (and these run on a 34 emulator), this attribute needs to be set.